.test-defs/pod-e2e.yaml

```diff
@@ -15,4 +15,4 @@ spec:
     ACCESS_KEY_ID_FILE=<(echo $ACCESS_KEY_ID)
     SECRET_ACCESS_KEY_FILE=<(echo $SECRET_ACCESS_KEY)
     IMAGE_TAG=$IMAGE_TAG
-  image: eu.gcr.io/gardener-project/3rd/golang:1.16.2
+  image: golang:1.16.2
```

Comment:
Basically same here: The testmachinery cluster will probably also run into docker hub rate limits if we do it like this, right?

Reply:
see above